README.md

```diff
@@ -256,7 +256,7 @@ let filters: BpfMap = seccompiler::compile_from_json(
 categories to BPF programs.
 
 ```rust
-pub type BpfMap = HashMap<String, Arc<BpfProgram>>;
+pub type BpfMap = HashMap<String, BpfProgram>;
```

Comment:
nit: can you add some details about what is changed in the documentation.

Reply:
I meant to add some details in the commit message, sorry about that.

Reply:
done